dev/reference/key-monitoring-metrics/tidb-dashboard.md

```diff
@@ -85,13 +122,21 @@ To understand the key metrics displayed on the TiDB dashboard, check the followi
     - Pseudo Estimation OPS: the number of the SQL statements optimized using pseudo statistics
     - Dump Feedback OPS: the number of stored statistical Feedbacks
     - Update Stats OPS: the statistics of using Feedback to update the statistics information
+    - Significant Feedback: the number of significant Feedback that update the statistics information
 
 - Meta
     - AutoID QPS: AutoID related statistics, including three operations (global ID allocation, a single table AutoID allocation, a single table AutoID Rebase)
     - AutoID Duration: the time consumed by AutoID related operations
+    - Meta Operations Duration 99: the latency of Meta operations
 
 - GC
     - Worker Action OPM: the statistics of GC related operations, including `run_job`, `resolve_lock`, and `delete\_range`
     - Duration 99: the time consumed by GC related operations
     - GC Failure OPM: the number of failed GC related operations
+    - Action Result OPM: the number of results of GC-related operations
     - Too Many Locks Error OPM: the number of the error that GC clears up too many locks
+
+- Batch Client
+    - Pending Request Count by TiKV: the number of Batch messages that are pending for processing
```

Comment:
It seems that the phrase `pending for` is incorrect or is seldom used. Ref: [Dictionary explanation](https://www.merriam-webster.com/dictionary/pending) and [discussion about this word](https://ell.stackexchange.com/questions/218701/is-pending-for-her-return-grammatical).

```suggestion
    - Pending Request Count by TiKV: the number of Batch messages that are pending processing
```